### ISSUE TITLE
BracesAroundHashParameters: ignore for single indent on multiple lines

### ISSUE THREAD

**Issue by ypresto** [bbatsov/rubocop#2139]
As ruby-style-guide, single indent is allowed for parameters with parentheses:

```ruby
# good (normal indent)
def send_mail(source)
  Mailer.deliver(
    to: 'bob@example.com',
    from: 'us@example.com',
    subject: 'Important message',
    body: source.text
  )
end
```

But currently rubocop warns about below braces around hash:

```ruby
desc 'Update API for something', {
  notes: 'Something will be changed by calling this API.'
}
```

How about ignoring braces for hash spans more than one line?

Thanks!

**Comment by jonas054** [comment]
I don't think so. Multi-line hashes are not that special. They're part of this cop's duties, so I think it would be a degradation in functionality to allow the braces.
